src/requiredIf.js

```diff
@@ -9,6 +9,7 @@ const requiredIfFactory = (condition, propType, isRequired) => (
     componentName,
     _location,
     propFullName // normally null but a string like "propName[index]" when wrapped in arrayOf
+    // eslint-disable-next-line max-params
```

Comment:
It should not be necessary to add that line. This repo has the following ESLint rule:
```javascript
'max-params': ['error', { max: 5 }],
```

Reply:
well, I couldn't commit anything without this one. Seems like something is going wrong on my local system...

Reply:
My local deps weren't up2date. I've removed that line